ext_emconf.php

```diff
@@ -10,28 +10,27 @@
  * writing. "version" and "dependencies" must not be touched!
  ***************************************************************/
 
-$EM_CONF[$_EXTKEY] = array (
-	'title' => 'Static Info Tables (cz)',
-	'description' => '(cz) language pack for the Static Info Tables providing localized names for countries.',
-	'category' => 'misc',
-	'version' => '1.0.1',
-	'state' => 'stable',
-	'uploadfolder' => 0,
-	'createDirs' => '',
-	'clearcacheonload' => 0,
-	'author' => 'Simon Schmidt',
-	'author_email' => 'sfs@marketing-factory.de',
-	'author_company' => 'Marketing Factory Consulting GmbH',
-	'constraints' => array(
-		'depends' => array(
-			'static_info_tables' => '6.0.0-6.2.99',
-			'php' => '5.3.0-5.6.99',
-			'typo3' => '6.2.0-6.2.99',
-		),
-		'conflicts' => array(
-		),
-		'suggests' => array(
-		),
-	),
-);
-
+$EM_CONF[$_EXTKEY] = [
+    'title' => 'Static Info Tables (cz)',
+    'description' => '(cz) language pack for the Static Info Tables providing localized names for countries.',
+    'category' => 'misc',
+    'version' => '6.3.0',
+    'state' => 'stable',
+    'uploadfolder' => 0,
+    'createDirs' => '',
+    'clearcacheonload' => 0,
+    'author' => 'Simon Schmidt',
+    'author_email' => 'sfs@marketing-factory.de',
+    'author_company' => 'Marketing Factory Consulting GmbH',
+    'constraints' => [
+        'depends' => [
+            'static_info_tables' => '6.0.0-6.3.99',
+            'php' => '5.3.0-5.6.99',
```

Comment:
I think, we should raise the dependency also to at least PHP 5.5, because short notation with "[]" instead of "array()" doesn't work in PHP 5.3.